cpp/include/Ice/Initialize.h

```diff
@@ -703,7 +703,7 @@ namespace IceInternal
     // to be used by modules such as Freeze.
```

Comment:
We should probably remove references to Freeze.

Reply:
Will remove in a separate PR.